core/node/node_monitoring_agent_tracker.go

```diff
@@ -53,3 +59,26 @@ func (m *MonitoringAgentTracker) Statuses() (MonitoringAgentStatuses, error) {
 
 	return MonitoringAgentStatuses{}, errors.New("identity not found")
 }
+
+// SessionItem represents information about session monitoring metrics.
+type SessionItem struct {
+	ID              string `json:"id"`
+	ConsumerCountry string `json:"consumer_country"`
+	Duration        int64  `json:"duration"`
+	StartedAt       int64  `json:"started_at"`
+	Earning         string `json:"earning"`
+	Transferred     int64  `json:"transferred"`
+}
+
+// MonitoringAgentSessions a object represent a sessions list of monitoring metrics.
```

Comment:
This is not related to the Monitoring Agent somehow. 
```suggestion
// ProviderSessions represents a list of provider sessions.
```